src/core/lexer.l

```diff
@@ -159,7 +162,10 @@ include[ \t\r\n]*"<"	{ BEGIN(cond_include); filepath = filename = ""; LOCATION_C
 
 use[ \t\r\n]*"<"		{ BEGIN(cond_use); LOCATION_COUNT_LINES(parserlloc, yytext); }
 <cond_use>{
-[\n\r]					{ LOCATION_ADD_LINES(parserlloc, yyleng); }
+[\n\r]					{
+							LOCATION_ADD_LINES(parserlloc, yyleng);
+							LOG(message_group::Warning,LOCATION(parserlloc),"","new lines 'use'-statement is not definded - behavior may change in the future");
```

Comment:
Typo `definded`. Maybe we could call it `use<>` to make it even clear what's meant?